src/Microsoft.AspNet.WebHooks.Receivers.Stripe/WebHooks/StripeWebHookReceiver.cs

```diff
@@ -2,81 +2,53 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Globalization;
 using System.Net;
 using System.Net.Http;
-using System.Net.Http.Headers;
+using System.Security.Cryptography;
 using System.Text;
 using System.Threading.Tasks;
 using System.Web.Http;
 using System.Web.Http.Controllers;
-using System.Web.Http.Dependencies;
-using Microsoft.AspNet.WebHooks.Config;
 using Microsoft.AspNet.WebHooks.Properties;
-using Newtonsoft.Json.Linq;
 
 namespace Microsoft.AspNet.WebHooks
 {
     /// <summary>
-    /// Provides an <see cref="IWebHookReceiver"/> implementation which supports WebHooks generated by Stripe. 
-    /// Set the '<c>MS_WebHookReceiverSecret_Stripe</c>' application setting to the application key defined in Stripe.
+    /// Provides an <see cref="IWebHookReceiver"/> implementation which supports WebHooks generated by Stripe.
+    /// Set the '<c>MS_WebHookReceiverSecret_Stripe</c>' application setting to the signing secret defined in Stripe.
     /// The corresponding WebHook URI is of the form '<c>https://&lt;host&gt;/api/webhooks/incoming/stripe/{id}</c>'.
-    /// For details about Stripe WebHooks, see <c>https://stripe.com/docs/webhooks</c>.
+    /// For details about Stripe WebHooks, see <c>https://stripe.com/docs/webhooks</c>. For details about Stripe
+    /// signing secrets, see <see href="https://stripe.com/docs/webhooks#signatures"/>.
     /// </summary>
-    public class StripeWebHookReceiver : WebHookReceiver, IDisposable
+    public class StripeWebHookReceiver : WebHookReceiver
     {
         // Application setting to enable test mode
         internal const string PassThroughTestEvents = "MS_WebHookStripePassThroughTestEvents";
 
-        // Do not verify the WebHook using a follow-up HTTP GET request but instead just use the WebHook as-is.
-        internal const string DirectWebHook = "MS_WebHookStripeDirect";
-
         internal const string RecName = "stripe";
```

Comment:
This probably deserves a better name

Reply:
I agree but&hellip; This is the pattern used in all full framework i.e. non-Core receivers.